movie/src/components/Icons/DotIcon/DotIcon.css

```diff
@@ -0,0 +1,21 @@
+.more-detail {
+  background-color: rgb(234, 244, 244);
+  width: 30px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 30px;
+  margin: 5px;
+  right: 2px;
+  position: absolute;
+}
+
+.more-detail:hover {
+  cursor: pointer;
+  background-color: rgb(151, 237, 237);
+}
+
+body:not(.modal-open){
```

Comment:
NOTED, for later. dont have to do this one

If you have time please find a better way of doing this